src/hooks/useProjects.ts

```
@@ -406,382 +412,273 @@ export const useProjects = () => {
+          subtasks: taskData.subtasks?.map((name, i) => ({ id: `${Date.now()}-${i}`, name, completed: false })),
           createdAt: new Date().toISOString(),
-          updatedAt: new Date().toISOString()
+          updatedAt: new Date().toISOString(),
         };
         setTasks(prev => [...prev, newTask]);
-        return newTask;
+        onTaskChange(); // Notify that project data might need a refresh
+        return;
       }
 
       const { data: taskResult, error: taskError } = await supabase
```

ESLint

[error] 'taskResult' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)